contracts/Tokens/Prime/PrimeStorage.sol

```diff
@@ -97,7 +97,7 @@ contract PrimeStorageV1 {
     /// @notice mapping used to find if an asset is part of prime markets
     mapping(address => address) public vTokenForAsset;
 
-    /// @notice Deprecated. Address of core pool comptroller contract
+    /// @notice Address of core pool comptroller contract
     address public comptroller;
```

Comment:
Would it make sense to rename it to an internal variable `corePoolComptroller` and add a public view `comptroller()` to maintain backward compatibility?